app/assets/stylesheets/application.css

```diff
@@ -71,6 +72,8 @@ th h3, th p {
     background-color: #fff;
 }
 
+about.countainer
```

Comment:
sorry that was the one line i forgot to remove. the new layouts page was added because the container in the original layout was adding padding to the about page and my teacher said that was the best way to fix it.

 
> On Jul 27, 2015, at 2:27 PM, Jason Shultz <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/application.css <https://github.com/stephyc93/att2.0/pull/113#discussion_r35582398>:
> 
> > @@ -71,6 +72,8 @@ th h3, th p {
> >      background-color: #fff;
> >  }
> >  
> > +about.countainer
> container is misspelled.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/stephyc93/att2.0/pull/113/files#r35582398>.
>